### ISSUE TITLE
Create embeddings offline from collected responses

### ISSUE THREAD

**Issue by stsievert** [stsievert/salmon#70]
Salmon's main goal is to minimize the number of responses required to obtain a good embedding. This requires creating an embedding but is not the primary goal.

For example, let's say I have 3 algorithms: TSTE, CKL and random sampling. How do I combine the responses from TSTE and CKL to create an embedding and test with random sampling?

**Comment by stsievert** [comment]
There is some work to help debug adaptive algorithms: https://docs.stsievert.com/salmon/developers.html#debugging, so this is definitely possible. I think most of this issue boils down to documentation on how to use the Scikit-Learn compatible estimators in `salmon.triplets.algs`. I think the interface will look something like this:

``` python
import pandas as pd
df = pd.read_csv("salmon-responses.csv")
X = df[["head", "left", "right"]].to_numpy()
y = df.apply(lambda row: 0 if row["left"] == row["winner"] else 1, axis=1).to_numpy()

from salmon.triplets.algs import TSTE
embed = TSTE()
embed.fit(X, y)
```

This will also require a bit of clean up.

**Comment by stsievert** [comment]
I think this is closed by #92.
